modules/gateway/persist.go

```diff
@@ -31,3 +33,13 @@ func (g *Gateway) load(filename string) (err error) {
 	}
 	return
 }
+
+// create logger
+// TODO: when is the logFile closed? Does it need to be closed?
```

Comment:
I think in this case it's actually fine to not close the file. It's standard practice to close resources *when you're done with them*, but we're never really "done with" logging until the program exits.